homework_4/tests/test_task_2_mock_input.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from task02.task_2_mock_input import count_dots_on_i
+
+
+@patch("urllib.request.urlopen")
+def test_count_dots_on_i(mock_urlopen):
+    mock = Mock()
+    mock.read.return_value = b"illustrative example"
+    mock_urlopen.return_value = mock
```

Comment:
you can set return value something like this: @patch('Foo.uses_some_other_method', return_value=Mock(read=Mock(return_value="specific_value"))):